pkg/controller/sparkapplication/monitoring_config.go

```diff
@@ -44,14 +44,10 @@ func configPrometheusMonitoring(app *v1beta2.SparkApplication, kubeClient client
 		port = *app.Spec.Monitoring.Prometheus.Port
 	}
 
-	var javaOption string
-	if app.HasPrometheusConfigFile() {
-		configFile := *app.Spec.Monitoring.Prometheus.ConfigFile
-		glog.V(2).Infof("Overriding the default Prometheus configuration with config file %s in the Spark image.", configFile)
-		javaOption = fmt.Sprintf("-javaagent:%s=%d:%s", app.Spec.Monitoring.Prometheus.JmxExporterJar,
-			port, configFile)
-	} else {
-		glog.V(2).Infof("Using the default Prometheus configuration.")
+	// If one of the metricsPropertiesFile & prometheusConfigFile is not set
+	// mounting configMap to using default configuration
+	if !app.HasMetricsPropertiesFile() || !app.HasPrometheusConfigFile() {
+		glog.V(2).Infof("Loading Prometheus configuration.")
```

Comment:
Can we change this to `Creating a ConfigMap for metrics and Prometheus configurations`?